packages/styles/package.json

```diff
@@ -44,6 +44,7 @@
   },
   "dependencies": {
     "@popperjs/core": "2.11.8",
+    "@swisspost/design-system-tokens": "workspace:*",
```

Comment:
Should we implement the correct version here instead of using an asterisk?

Reply:
This is how workspace dependencies get installed by pnpm when using the `--workspace` flag. I'm wondering if we should change all other workspace dependencies to that.